python/templates/DatamodelDefinition.h.jinja2

```diff
@@ -0,0 +1,30 @@
+// AUTOMATICALLY GENERATED FILE - DO NOT EDIT
+
+#include "podio/EDMDefinitionRegistry.h"
+
+namespace {{ package_name }}::meta {
+/**
+ * The complete definition of the datamodel at generation time in JSON format.
+ */
+static constexpr auto {{ package_name }}__JSONDefinition = R"EDMDEFINITION({{ edm_definition }})EDMDEFINITION";
```

Comment:
The handling of referenced-to types not present in the file is incomplete. As of now we would end up with not consistent yaml files if type B (pointing to A) is in the file, while type A is missing.
We need to find a way for how to insert it nicely.